docs/harbor_backup_restore.md

```diff
@@ -0,0 +1,247 @@
+# Harbor - backup and restore
+
+This page aims at providing a step-by-step guide for backup and restore [Harbor](https://goharbor.io/)
+container registry using [Velero](https://velero.io/) tool.
+It extends the official [Harbor backup-restore docs page](https://goharbor.io/docs/main/administration/backup-restore/)
+with up-to-date commands, explanations, and an extensive prerequisites section. This
+guide references and uses Velero in [v1.10.2](https://github.com/vmware-tanzu/velero/releases/tag/v1.10.2)
+as this is the latest stable version at the time of writing this guide.
```

Comment:
> this is the latest stable version at the time of writing this guide

I like this disclaimer.

In the general context of maintenance: Is it the goal to provide docs which are "future-proof" or is it okay to serve as a reference of "this worked at the time of writing"? @maxwolfs ?

(When thinking about future-proofing documentation, I'd land at continuously "unittesting" documentation and using e.g. renovate to keep it up to date. As renovate/CI-testing has barely landed in the more code-centric contexts in SCS, this of course would be a topic for some time in the future - not for now anyway.)

Not a blocking problem for this PR anyway.

Reply:
The idea of future-proofing documentation with a continuous "unittesting" pipeline is interesting from my point of view.   IMO, it may require a serious effort for writing CI pipelines that should cover possibly various scenarios that could be mentioned in the docs.
   
@maxwolfs, it would be great to know your opinion here.

Reply:
Thanks for bringing this up. I think this is a good topic to discuss further within the next SIG Documentation on the 15th of May at 11:05 at SCS-Tech. 

In the [2023-02-20](https://github.com/SovereignCloudStack/minutes/blob/main/sig-documentation/20230220.md#version-current-docs-for-upcoming-release) date we discussed versioning and came to the conclusion that we only want to serve _latest_ documentation.

From that point of view, I expect that the docs will always work and if they do not because of anything stops working, code & docs have to be updated. Then `at the time of writing this guide` is somehow obsolete from my point of view because it applies for the whole docs, but makes of course sense in combination with mentioning specific versions. 

I don't see this as a blocker too. Let's merge :)

Reply:
@joshmue could we resolve this one?

Reply:
Sure!

(even if the PR could be merged either way, of course)